utils/scarb-stable-hash/src/lib.rs

```diff
@@ -7,6 +7,7 @@ use xxhash_rust::xxh3::Xxh3;
 ///
 /// The hasher should be fast and have a low chance of collisions (but is not sufficient for
 /// cryptographic purposes).
+#[derive(Default)]
```

Comment:
if we derive Default, then let's `StableHasher::new` use it

it amazes me that clippy didn't catch this